src/content/guides/fine-tuning-upgrade-disruption-on-aws/index.md

```diff
@@ -1,123 +1,136 @@
+Configurability of these details has been introduced with Giant Swarm release v12.7.0 for AWS. The feature is currently in an early stage and its behaviour may change in the near future.
 
-There are two valid formats for the `MaxBatchSize`:
+Adjustments to these settings require using the [Control Plane Kubernetes API](/basics/api/#cp-k8s-api) to edit the [`AWSCluster`](/reference/cp-k8s-api/awsclusters.infrastructure.giantswarm.io/) resource of the cluster (for cluster-wide settings) or the [`AWSMachineDeployment`](/reference/cp-k8s-api/awsmachinedeployments.infrastructure.giantswarm.io/) of an individual node pool.
 
-* An integer number bigger than zero, which specifies the static amount of nodes, e.g "5"
-* A decimal number in a range `0 < x <= 1.0` which defines a percentage of nodes that can be rolled in a single batch, e. g. "0.25" which represents 25% of all nodes
+## Maximum batch size {#maximum-batch-size}
 
-The default value is `0.3` which implies 30% of instances will be rolled in each batch.
+When a worker node update is necessary, nodes are updated (terminated and replaced by new ones) in groups or batches. The maximum batch size configures how many nodes are updated at the same time.
 
-### MaxBatchSize examples
+The settings directly maps to the [`MaxBatchSize`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-attribute-updatepolicy.html#cfn-attributes-updatepolicy-rollingupdate-maxbatchsize) property of the AWS CloudFormation update policy.
 
-In this example, we set value to static value `10` to roll maximum of 10 instances per single batch.
+The **default value** for this property is `0.3`, which means that each batch will contain at most 30% of the nodes.
 
-```yaml
+In order to override the default for either the entire cluster or a spdcific node pool, the annotation
+
+```nohighlight
+alpha.aws.giantswarm.io/update-max-batch-size
+```
+
+must be set
+
+- on the [`AWSCluster`](/reference/cp-k8s-api/awsclusters.infrastructure.giantswarm.io/) resource, to be applied as a default to **all node pools** of the cluster.
+- on the [`AWSMachineDeployment`](/reference/cp-k8s-api/awsmachinedeployments.infrastructure.giantswarm.io/) resource, to be effective for only **one node pool**. A value here will override any value specified on the `AWSCluster` level.
+
+You have two options to configure the maximum batch size:
+
+- **Absolute**: using an integer number larger than zero, you'll specify the maximum number of nodes in absolute terms.
+- **Relative**: using a decimal number between `0.0` and `1.0` you can define the group size as a percentage of the node pool size. As an example, the value `"0.25"` would mean that all worker nodes would be divided into four groups, each containing 25 percent of the worker nodes (roughly).
```

Comment:
maybe it's worth mentioning that they cannot use `0` but they can use `1.0`  (as 100% makes sense  and 0% dont)

you cannot roll zero nodes so the minimum number will always be 1 node